eox_nelp/apps.py

```diff
@@ -33,9 +28,27 @@ class eoxNelpConfig(AppConfig):
                 'production': {'relative_path': 'settings.production'},
                 'devstack': {'relative_path': 'settings.devstack'},
             },
+        }
+    }
+
+
+class EoxNelpCMSConfig(AppConfig):
+    """App configuration"""
+    name = 'eox_nelp'
+    verbose_name = "eduNEXT Openedx Extensions"
```

Comment:
I'm not sure about this

Reply:
@andrey-canon what is the problem? THe name???

Reply:
this could be Nelp Openedx Extension since "eduNEXT Openedx Extensions" is eox-core

Reply:
Ah ok heheh let me I change it.

Reply:
Resolved in https://github.com/eduNEXT/eox-nelp/pull/2/commits/a125a7569bbfd536f28897a1a2c94aa41e39a324